### PR TITLE
Turn unique feature into implicit ad-hoc containers

### PR DESCRIPTION
**Pull request by michaelsauter**
Addresses #265.

### CODE REVIEW

File: crane/container.go [hunk 1]
```diff
@@ -1050,12 +1048,16 @@ func (c *container) createArgs(cmds []string, excluded []string) []string {
...
+	if !adHoc {
+		for _, port := range c.RunParams().Publish() {
+			args = append(args, "--publish", port)
+		}
 	}
 	// PublishAll
-	if c.RunParams().PublishAll {
-		args = append(args, "--publish-all")
+	if !adHoc {
+		if c.RunParams().PublishAll {
```
Comment (bjaglin) [inline crane/container.go]: combine in one if ?

File: crane/container.go [hunk 2]
```diff
@@ -933,8 +929,10 @@ func (c *container) createArgs(cmds []string, excluded []string) []string {
 		args = append(args, "--env-file", envFile)
 	}
 	// Expose
-	for _, expose := range c.RunParams().Expose() {
-		args = append(args, "--expose", expose)
+	if !adHoc {
```
Comment (bjaglin) [inline crane/container.go]: I can't see any collision with `expose`, so why blacklist it for adhoc containers?

File: README.md [hunk 3]
```diff
@@ -304,6 +303,15 @@ This feature is experimental, which means it can be changed or even removed
 in every minor version update.
 
 
+### Ad hoc commands
+If you pass a command on the CLI to `lift` or `run`, Crane will add a timestamp
+to the container name (e.g. `foo` will become `foo-1447155694523`), making it
+possible to have multiple containers based on the same Crane config. Those ad
+hoc containers will have `expose`, `publish`, `publish-all` and `detach`
```
Comment (bjaglin) [inline README.md]: `--ip`, `--ip6` should also be ignored since they can collide

**Comment by bjaglin** [comment]
:+1:

File: crane/container.go (hunk 2 above)
Comment (michaelsauter) [inline crane/container.go]: Oh right. I just saw them mentioned in the CLI docs and thought they worked differently, haven't used that option so far. Now after properly reading the doc I see it's fine. Will undo.

File: crane/container.go (hunk 1 above)
Comment (michaelsauter) [inline crane/container.go]: Yea, thought it's nicer to read separately.

File: README.md (hunk 3 above)
Comment (michaelsauter) [inline README.md]: True! I'll add.

**Comment by michaelsauter** [comment]
I'll update and then give it a test run next week at work, if all goes well I'll merge then.

**Pull request merged**